src/OAM.md

```diff
@@ -53,7 +53,7 @@ tile is "NN & \$FE", and the bottom 8×8 tile is "NN | \$01".
   "Attributes"  7:"Priority" 6:"Y flip" 5:"X flip" 4:"DMG palette" 3:"Bank" 2-0:"CGB palette";
 }}
 
-- **Priority**: `0` = No, `1` = BG and Window colors 1–3 are drawn over this OBJ
+- **Priority**: `0` = No, `1` = BG and Window [color IDs](./Tile_Data.md) 1–3 are drawn over this OBJ
```

Comment:
```suggestion
- **Priority**: `0` = No, `1` = BG and Window [color indices](./Tile_Data.md) 1–3 are drawn over this OBJ
```
[The priority description](https://gbdev.io/pandocs/Tile_Maps.html#bg-to-obj-priority-in-cgb-mode) calls them “color index”es, which I think is a little clearer. (Is the term “color ID” used anywhere else? We should probably change it there too, if so.)

Reply:
Yes please!